database/migrations/2023_01_13_000005_correct_inefficient_auditkeys.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+use Illuminate\Database\Eloquent\Collection;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+use Sourcetoad\Logger\Helpers\DataArrayParser;
+use Sourcetoad\Logger\Models\AuditKey;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        AuditKey::query()->chunkById(1000, function (Collection $auditKeys) {
```

Comment:
Any chance we run into large data sets better served by using `cursor`?

Reply:
100k is most and I peeked this thread: https://stackoverflow.com/a/53736557 and thought 1000 chunking looked the best.

Reply:
Eh I guess thats the point of chunking - nvm.